packages/module/patternfly-docs/content/extensions/virtual-assistant/design-guidelines.md

```diff
@@ -0,0 +1,206 @@
+![2 toggles with open and closed icons.](./img/chatbot-toggle.svg)
+</div>
+
+When there is an unread message from the ChatBot, a notification badge should be placed on the toggle.
+
+<div class="ws-docs-content-img">
+![Toggle with notification badge.](./img/chatbot-toggle-notification.svg)
+</div>
+
+### Using the navigation menu
+
+The ChatBot navigation menu primarily contains a users' conversation history with the ChatBot. Clicking the menu icon opens a side drawer in the ChatBot window. 
+
+By clicking into the navigation menu, users can search through previous conversations and perform additional actions, such as sharing a conversation with others. 
+
+<div class="ws-docs-content-img">
+![Conversation history with an options menu opened on a previous conversation.](./img/conversation-history.svg)
+</div>
+
+### Attaching files 
+
+Using [the attach button](/patternfly-ai/chatbot/overview/design-guidelines#message-bar) in the message bar, users can [attach files](/patternfly-ai/chatbot/messages#file-attachments) to their message to share with the ChatBot. 
+
+The attach button can follow a couple of patterns, including: 
+- Opening the file explorer for a user's operating system
+- Opening a menu with attachment options that are chosen by designers and developers
+
+<div class="ws-docs-content-img">
+![Menu with file attachment options.](./img/attachment-menu.svg)
+</div>
+
+If a message attachment is successful, a label with the file details will be displayed in the message:
+
+<div class="ws-docs-content-img">
+![File attached to user message.](./img/attached-file.svg)
+</div>
+
+Users can select the file label to either preview or edit their attachment, as shown in these [attachment examples](/patternfly-ai/chatbot/messages#attachment-preview).
+
+If a message attachment fails, an error message should share the reason for failure:
+
+<div class="ws-docs-content-img">
+![Error alert for failed attachment.](./img/attachment-error.svg)
+</div>
+
+## Variations 
+
+### Display modes
+
+There are a few display modes that users can choose when interacting with a ChatBot.
+
+<div class="ws-docs-content-img">
+![Menu of display options.](./img/display-menu.svg)
+</div>
+
+1. **Overlay:** The default display mode, which floats the ChatBot window on top of a product's UI. In overlay mode, the ChatBot can be opened and minimized with the toggle. 
+
+<div class="ws-docs-content-img">
+![ChatBot in overlay mode.](./img/overlay.svg)
+</div>
+
+2. **Docked:** Anchors the ChatBot to the side of the page content. When docked, the ChatBot window is persistent, and cannot be toggled.
+
+<div class="ws-docs-content-img">
+![ChatBot in docked mode.](./img/docked.svg)
+</div>
+
+3. **Full screen:** Fills the screen with the ChatBot window.
+
+<div class="ws-docs-content-img">
+![ChatBot in full-screen mode.](./img/fullscreen.svg)
+</div>
+
+4. **Embedded:** Places the ChatBot within a product as its own page with its own item navigation menu.
```

Comment:
It doesn't have to have its own navigation menu - technically they can use any part of the chatbot in this (just the message box and input, etc.)

Reply:
I think that's accurate!